.env.example

```
@@ -1,47 +1,71 @@
-UPLOAD_MODE=local
-UPLOAD_MAX_MB=20
-UPLOAD_ALLOWED_EXT=pdf,png,jpg,jpeg,webp,doc,docx
-UPLOAD_ALLOWED_MIME=application/pdf,image/png,image/jpeg,image/webp,application/msword,application/vnd.openxmlformats-officedocument.wordprocessingml.document
 
-# AWS S3 (if using cloud storage)
-AWS_ACCESS_KEY_ID=your_aws_access_key
-AWS_SECRET_ACCESS_KEY=your_aws_secret_key
-AWS_REGION=us-east-1
-AWS_S3_BUCKET=your_s3_bucket
+# File Upload Configuration
+UPLOAD_MODE=r2  # local or r2
```

dotenv-linter (3.3.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)